apps/codeforafrica/src/components/ErrorHero/ErrorHero.js

```diff
@@ -0,0 +1,65 @@
+import { RichTypography, Section } from "@commons-ui/core";
+import Grid from "@mui/material/Grid";
+import React from "react";
+
+import TwoToneBackground from "@/codeforafrica/components/TwoToneBackground";
+
+const ErrorHero = React.forwardRef(function ErrorHero(props, ref) {
+  const { title, subtitle, sx, tags, ...other } = props;
+
+  return (
+    <TwoToneBackground
+      sx={{ py: { xs: "43px", sm: 0, md: "65px", lg: 0 } }}
+      {...other}
+      ref={ref}
+    >
+      <Section
+        sx={{
+          maxWidth: {
+            sm: "648px",
+            md: "912px",
+          },
+          px: { xs: 2.5, sm: 0 },
+          py: { xs: 5, md: "25px" },
+          zIndex: 1,
+          ...sx,
+        }}
+      >
```

Comment:
1. Using `py` on either `TwoToneBackground` or `Section` **only** (instead of both) will simplify things. Otherwise, one has to add them up to know what's the total padding from top/bottom of header to a component.
2. Looking at text only headers such as the one in Opportunities, `py` for `ErrorHero` should be around `{ xs: 7.5, md: 10, lg: 12.5 }` (lg is in designs).